test/parallel_exec.jl

```diff
@@ -820,7 +821,12 @@ if DoFullTest
     end
     sleep(0.5)  # Give some time for the above error to be printed
 
-@unix_only begin
+    # github PR #14456
+    for n = 1:10^6
+        fetch(@spawnat myid() myid())
```

Comment:
I think this was just git rebase confusion. I was trying to remove the `unix_only`, and missed that the whole function had moved